release/202001/metrics-head-rc.md

```diff
@@ -5,8 +5,9 @@
 
 ## Focus Areas by Working Group  
 #### Common Metrics WG  
-* [When: Time](#user-content-focus-area---when-time)  
-* [Who: People](#user-content-focus-area---who-people)  
+* [What](#user-content-focus-area---what)  
```

Comment:
How about keeping the focus area names unchanged until the regular release? We would just add the new focus area.

Reply:
Yeah, I thought about that as well. But once we make any changes at all to the release page, the integrity of the release is questionable. With regular releases, the page will change constantly. I already need to pull an evolution metric in so it is going to change again. The PDF is now the static record. In regard to the name change itself I made it to match the language in the common repo.